pkgs/tools/archivers/7zz/default.nix

```diff
@@ -2,23 +2,27 @@
 , lib
 , fetchurl
 
+  # Only used for x86/x86_64
```

Comment:
It is very likely that the comment will be forgotten to be updated when something update the condition changes.

Reply:
It is very unlikely that this will ever change though. `uasm` only builds x86 and x86_64 assembly, and upstream said they're not interested of adding any new backends.

Also, even if `uasm` added support for, say, `aarch64`, I don't think `7zz` would switch to it since they already wrote the code in `.S` format.